packages/core/src/dispatcher.ts

```diff
@@ -16,7 +16,8 @@ import { Subscription } from 'rxjs'
 import { MessageBus } from './pubsub/message-bus'
 import { LRUMap } from 'lru_map'
 
-const IPFS_GET_TIMEOUT = 60000 // 1 minute
+const IPFS_GET_RETRIES = 3
+const IPFS_GET_TIMEOUT = 30000 // 30 seconds per retry, 3 retries = 90 seconds total timeout
```

Comment:
The timeout should probably be ~3 seconds instead of 30 seconds since this could be called during user API processing and normally a couple of seconds is considered the maximum a user should have to wait for an API call to return.

What are your thoughts?

Reply:
I think 3 seconds would be way too short when we're thinking about navigating the ipfs DHT to do content lookups

Reply:
I'd rather err on the side of keeping the timeouts longer than needed for now, rather than risk timing out operations that might have succeeded if they were given more time.